### DISCUSSION TITLE
How does one force suricata to use PF_RING?

### DISCUSSION THREAD

**Discussion by sempervictus** [Security-Onion-Solutions/securityonion#8411] [Unsupported Versions] [answered]
In the days of old, SO supported use of PF_RING. We're such fans of this that we integrated PF_RING builds into the kernel tree (the scriptage to do this was accepted into the PF_RING tree).
Suricata still supports this, but the way Salt deploys and runs suricata in the container forces af-packet at the commandline.
For those of us who still have our own optimized builds of PF_RING readily available on our systems, how do we enable the functionality in SO?

**Comment by dougburks** [comment, marked as answer]
We haven't supported PF_RING since Security Onion 16.04 (now past End Of Life).
